docs/04. Guards.md

```diff
@@ -278,7 +278,7 @@ You can also specify optional actions, so that the rule only apply to one or sev
 
 ```php
 return [
-    'zfc_rbac' => [
+    'lmc_rbac' => [
         'guards' => [
             'ZfcRbac\Guard\ControllerGuard' => [
```

Comment:
Still Zfc here.

Reply:
hmm. docs are outdated anyway. but i'll change it